http/src/test/java/org/mule/test/module/http/functional/requester/HttpsRequestNtlmProxyTestCase.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.test.module.http.functional.requester;
+
+import static javax.servlet.http.HttpServletResponse.SC_OK;
+import static javax.servlet.http.HttpServletResponse.SC_PROXY_AUTHENTICATION_REQUIRED;
+import static org.glassfish.grizzly.http.server.Constants.CLOSE;
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.junit.Assert.assertThat;
+import static org.glassfish.grizzly.http.server.Constants.CONNECTION;
+import static org.mule.runtime.http.api.HttpHeaders.Names.PROXY_AUTHENTICATE;
+import static org.mule.runtime.http.api.HttpHeaders.Names.PROXY_AUTHORIZATION;
+import static org.mule.test.allure.AllureConstants.HttpFeature.HTTP_EXTENSION;
+import static org.mule.test.module.http.functional.matcher.HttpMessageAttributesMatchers.hasStatusCode;
+
+import java.io.IOException;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.eclipse.jetty.proxy.ConnectHandler;
+import org.eclipse.jetty.server.Request;
+import org.eclipse.jetty.server.Server;
+import org.eclipse.jetty.server.handler.AbstractHandler;
+import org.junit.Test;
+import org.mule.extension.http.api.HttpResponseAttributes;
+import org.mule.runtime.api.message.Message;
+
+import ru.yandex.qatools.allure.annotations.Features;
+
+/**
```

Comment:
Here you can and should use Allure annotations. Features, Stories and Description in this case, see other tests for examples.